### PR TITLE
Adding in ability to exclude namespaces:

### PR DESCRIPTION
**Pull request by minleychris**

### CODE REVIEW

**Comment by Idorobots** [comment]
+1

**Comment by skazhy** [comment]
+1 This would be really useful. Just ran into the issue #47 and this looks like the best workaround for now.

File: cloverage/src/cloverage/coverage.clj [hunk 1]
```diff
@@ -131,10 +135,14 @@
         add-test-nses (:extra-test-ns opts)
         ns-regexs     (map re-pattern (:ns-regexp opts))
         test-regexs   (map re-pattern (:test-ns-regexp opts))
+        exclude-regex (map re-pattern (:ns-exclude-regex opts))
         start         (System/currentTimeMillis)
         test-nses     (concat add-test-nses (find-nses test-regexs))
-        namespaces    (concat add-nses      (find-nses ns-regexs))
-        ]
+        namespaces    (clojure.set/difference
```
Comment (palfrey) [inline cloverage/src/cloverage/coverage.clj]: Given clojure.set is explicitly imported, can this please become 'set' not 'clojure.set'?

**Comment by palfrey** [comment]
I'm pretty happy with this, with two minor items if you wouldn't mind fixing them:
- The clojure.set/set item above
- Can you replace the "namespaces" bit in "loading namespaces" with (apply list namespaces) so that the existing behaviour of printing as a list is preserved?

**Comment by palfrey** [comment]
This will need current master re-merging into it before I can merge it, as well as the items mentioned above

**Comment by MichaelBlume** [comment]
How's this? https://github.com/MichaelBlume/cloverage/tree/exclude_nses

**Comment by palfrey** [comment]
@MichaelBlume There's also #71, but I prefer the way you've done it. If you can open up a pull request for that, I'll merge it in and get all of these other ones sorted out.

**Comment by MichaelBlume** [comment]
Done =)

**Pull request merged**